Run7/step01/flats.cfg

```diff
@@ -0,0 +1,66 @@
+[ACQUIRE]
+flat
+
+[FLAT]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
+BCOUNT=1        # number of bias frames per flat image
+WL    =red    # wavelength filter to use for the flats
+HILIM =15.0    # maximum seconds for a flat field exposure
+LOLIM =0        # minimum seconds for a flat field exposure
+FILTERCONFIG = /gpfs/slac/lsst/fs2/u1/devel/ccs/bot-eotest-configs/Run6/calib.cfg
+
+DARKINTERRUPT = False # If True, will take darks between flat images 
+DARKINTERRUPTLIST = 30  5 # integration time and image count for dark set, formatted the same as the DARK acquisition
```

Comment:
It would be good to have your PR for handling these entries in fp-scripts reviewed and merged so that we can test this functionality ahead of time.